src/features/book/dto/create-book.dto.ts

```diff
@@ -1 +1,13 @@
-export class CreateBookDto {}
+/* eslint-disable prettier/prettier */
```

Comment:
Why we disable prettier here ?

Reply:
To exclude files from formatting.
To make the code look better

Reply:
Everything that you described above you disabled with this line :)